### PR TITLE
Project Labyrinth by Pauline Andersson & Ann-Sofi Jönsson 

### PR DESCRIPTION
**Pull request by Ann-Sofi**

### CODE REVIEW

**Review by darcywinterhill — APPROVED** [review]
Overall a fun, clean project that's easy to navigate through! 🎉👌🏼

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
After I click the start button I saw this page: 
image
I'm really hungry for adventure so please fix it and let me play ⚔️

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,52 @@
-import React from 'react'
+import React from 'react';
+import { Provider } from 'react-redux';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import styled from 'styled-components/macro';
+
+import labyrinth from './reducers/labyrinth'
```
Comment (KarolCislowski) [inline code/src/App.js:6]: Check your Prettier config to make it add/remove all semicolons for you 😸

File: code/src/components/GameScreen.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+  const setIcon = () => {
+    switch(data.coordinates){
+      case "0,1": return <span style={{fontSize: "40px"}} role="img" aria-label="robot">🤖</span >
+      case "0,2": return <span style={{fontSize: "40px"}} role="img" aria-label="music">🎵</span >
+      case "0,3": return <span style={{fontSize: "40px"}} role="img" aria-label="scroll">📜</span>
+      case "1,0": return <span style={{fontSize: "40px"}} role="img" aria-label="star">✨</span>
+      case "1,1": return <i className="nes-icon is-medium heart"></i>
+      case "1,3": return <i className="nes-icon trophy is-large"></i>
+      default: return <i className="nes-icon is-medium star"></i>
```
Comment (KarolCislowski) [inline code/src/components/GameScreen.js:49]: Nice :)

File: code/src/components/GameScreen.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            flexDirection: "column",
+            justifyContent: "space-between"}}>
+          <p style={{fontSize: "14px"}}>{button.description}</p>
+          <button type="button"
+            className="nes-btn"
+            style={{marginTop: "10px", width: "fit-content"}}
+            value={button.direction}
+            onClick={(e) => onChooseDirection(e)}>
+            Go {button.direction}
+          </button></div>)
```
Comment (KarolCislowski) [inline code/src/components/GameScreen.js:85]: If you mapping something it is a good practice to create a separate Component and return it here. It is easier to read and debug 🤓

File: code/src/components/StartScreen.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+            value={userInput}
+            onChange={(e)=> setUserInput(e.target.value)} />
+          </div>
+        <button
+          type="button"
+          style={{marginTop: "20px"}}
+          className="nes-btn"
+          onClick={onUsernameSet}>
+            Start the game!
+        </button>
```
Comment (KarolCislowski) [inline code/src/components/StartScreen.js:76]: It is another example of a great opportunity to create a separate component 😸

**Review by KarolCislowski — APPROVED** [review]
After a deep analysis of your code, I found out that the issue I pointed before was closed by API, not by error inside your code. I sent a memo about it to teachers and I need to sorry for any inconvenience done because of it 😢

**Pull request closed without merging**